googlemock/src/gmock-spec-builders.cc

```diff
@@ -487,8 +487,24 @@ bool UntypedFunctionMockerBase::VerifyAndClearExpectationsLocked()
       // takes care of it.
       untyped_expectation->MaybeDescribeExtraMatcherTo(&ss);
       untyped_expectation->DescribeCallCountTo(&ss);
-      Expect(false, untyped_expectation->file(),
-             untyped_expectation->line(), ss.str());
+      if (! registry_destructed_) {
```

Comment:
The if (registry_destructed_) case can go first, with a return.
Then you don't change the Expect(false,... ); statement.